makefiles/dependencies/kahip.mk

```diff
@@ -0,0 +1,37 @@
+KAHIP_DEF ?= -DHAVE_KAHIP
+
+ifdef KAHIP_INSTALL_PATH
+  KAHIP_INC_PATH := -I$(KAHIP_INSTALL_PATH)/parallel/parallel_src/interface \
```

Comment:
This should just be `-I$(KAHIP_INSTALL_PATH)/include`
if the kahip library is properly "installed" i.e. during compilation you have `-DCMAKE_INSTALL_PREFIX=somedir` then during OP2 make config you have `KAHIP_INSTALL_PATH=somedir`